src/main/java/com/cegeka/academy/web/rest/errors/EmptyChallengeSetException.java

```diff
@@ -0,0 +1,15 @@
+package com.cegeka.academy.web.rest.errors;
+
+public class EmptyChallengeSetException extends Exception {
```

Comment:
Numele nu este tocmai potrivit. In orice caz, in momentul in care nu se gaseste o resursa se arunca intotdeauna NotFound indiferent de tipul resurselor cautate. Prin urmare, aceasta exceptie nu este necesare. Poti arunca NotFoundException.